backend/src/apps/api/v1/organizations.py

```diff
@@ -21,15 +36,25 @@ def perform_create(self, serializer):
         )
 
 
-class OrganizationDetailsAPI(generics.RetrieveUpdateDestroyAPIView):
+@extend_schema(tags=["Организации"])
+@extend_schema_view(
+    patch=extend_schema(
+        summary="Обновление организации",
+        request=OrganizationSerializer,
+        responses={status.HTTP_200_OK: OrganizationSerializer},
+    ),
+    get=extend_schema(
+        summary="Получение информации об организации",
+        responses={status.HTTP_200_OK: OrganizationSerializer},
+    ),
+    delete=extend_schema(
+        summary="Удаление организации",
+        responses={status.HTTP_204_NO_CONTENT: None},
+    ),
+)
+class OrganizationDetailsAPI(BaseOrganizationDetailView):
+    queryset = Organization.objects.filter(membership__user=models.OuterRef('user'))
```

Comment:
Аналогично предыдущему
```suggestion
class OrganizationDetailsAPI(generics.RetrieveUpdateDestroyAPIView):
```